src/collection-view.js

```diff
@@ -522,6 +545,18 @@ Marionette.CollectionView = Marionette.View.extend({
     return childViews;
   },
 
+  // Return true if the given child should be shown
+  // Return false otherwise
+  // The filter will be passed (child, index, collection)
+  // Where
+  //  'child' is the given model
+  //  'index' is the index of that model in the collection
+  //  'collection' is the collection referenced by this CollectionView
+  shouldAddChild: function (child, index) {
+    var filter = this.getOption('filter');
+    return !_.isFunction(filter) || filter.call(this, child, index, this.collection);
+  },
```

Comment:
yeah I think it should be, no need to expose this method since we already give people the `filter` method to set